docs/rules/no-universal-selectors.md

```diff
@@ -0,0 +1,30 @@
+# No Universal Selectors
+
+Rule `no-universal-selectors` will enforce that the `*` (universal) selector is not allowed to be used.
```

Comment:
This doesn't read right, Will enforce the use of universal selectors or enforce whether universal selectors can be used.. Something more like that?